packages/dx-react-scheduler/docs/reference/day-view.md

```diff
@@ -26,7 +26,8 @@ none
 
 Name | Type | Default | Description
 -----|------|---------|------------
-name? | string | `Day` | The view name. Required if you use several `DayView` plugins.
+name? | string | `Day` | The view unique identifier. Required if you use several `DayView` plugins.
+displayName? | string |  | The view name used in UI plugins. Default value is equal to `name`.
 intervalCount? | number | 1 | Multiplies the default view interval.
 cellDuration? | number | 30 | Specifies the cell duration in minutes.
```

Comment:
```suggestion
cellDuration? | number | 30 | Specifies the cell's duration in minutes.
```

Reply:
BTW, how can a cell have a duration?

Reply:
This is a Scheduler, one cell ~ 30 minutes by default.